lib/curl_multibyte.c

```diff
@@ -82,6 +82,21 @@ char *curlx_convert_wchar_to_UTF8(const wchar_t *str_w)
 
 #if defined(USE_WIN32_LARGE_FILES) || defined(USE_WIN32_SMALL_FILES)
 
+int curlx_win32_open(const char *filename, int oflag)
+{
+#ifdef _UNICODE
+  int result = 0;
+  wchar_t *filename_w = curlx_convert_UTF8_to_wchar(filename);
+  if(filename_w)
+    result = _wopen(filename_w, oflag);
+  free(filename_w);
+  if(result)
+    return result;
+#endif
+
+  return (open)(filename, oflag);
```

Comment:
Why even define the function on non Windows platforms?

Reply:
It's not, this is in a `#if defined(USE_WIN32_LARGE_FILES) || defined(USE_WIN32_SMALL_FILES)` block.